packages/ui/v2/core/Shell.tsx

```diff
@@ -183,6 +183,8 @@ type LayoutProps = {
   withoutMain?: boolean;
   // Gives you the option to skip HeadSEO and render your own.
   withoutSeo?: boolean;
+  // Gives the ability to include actions to the right of the heading
+  actions?: JSX.Element;
```

Comment:
Adding the ability to include actions to the right of the title/subtitle section of the shell, where the search input gets added